packages/react-devtools-shared/src/backend/index.js

```diff
@@ -94,7 +95,27 @@ export function initBackend(
     }
   });
 
+  const getIsReloadAndProfileSupported =
+    hook.settings?.getIsReloadAndProfileSupported ??
```

Comment:
This won't do anything, because you don't store it on the `Hook` object, and you don't need to.

Can you add boolean optional parameter to `initBackend` here?
https://github.com/facebook/react/blob/a15bbe14751287cb7ac124ff88f694d0883f3ac6/packages/react-devtools-shared/src/backend/index.js#L16-L20

Then, anywhere where you are calling `initBackend`: browser extension content script, `react-devtools-core`, `react-devtools-inline`, you will provide this argument.

For browser extension it will be `true` by default, for` react-devtools-inline` it will be `false` (we don't support it), for `react-devtools-core` this will be the return of the function `isReloadAndProfileSupported`, which is passed from the Host (React Native in our case)